.gitignore

```diff
@@ -44,7 +44,7 @@ tmp
 
 # Binaries
 /bin/
-__debug_bin
+__debug_bin*
```

Comment:
Go started building multiple debug_bin files, the asterisk ensures nothing gets committed.